docker/linux-x86_64/v8_node.Dockerfile

```diff
@@ -0,0 +1,40 @@
+# Copyright (c) 2021 caoccao.com Sam Cao
+# All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Usage: docker build \
+#  -t amithgeorge/javet-linux-dev:v8-$JAVET_V8_VERSION_node-$JAVET_NODE_VERSION \
+#  -f docker/linux-x86_64/base-node.Dockerfile .
+
+ARG JAVET_V8_VERSION
```

Comment:
I am not sure what the concern is. The older images (for v8, node) once built, are present on DockerHub and wouldn't need to be built again. 

The source of truth for the version numbers (of V8, Node) is the GitHub workflow files. I would image if you need to do a bug fix for an older version, you might 
- checkout that tag
- fix the code
- the workflow file still contains the node and v8 version relevant at that time. 
- push a new tag
- the workflow should run for this new tagged commit and that workflow should use the version numbers present in the file. 

The above is my understanding. If I have misunderstood something, please correct me. I haven't actually done the above steps before, so I don't know for a 100% if it will work. Do you think we can test it out?

Reply:
The steps you described belong to an ideal one which covers most of the cases.

However, there are some edge cases. E.g.

- CMake needs to be upgraded.
- Python 2 and 3 conflict with each other.
- Patch scripts need to be upgraded.
- ...

These edge cases happen once per 2-3 releases. So, I'd like each layer to be sort of _versioned_ so that we keep the ability of replacing any layer in any legacy release. Hope that makes sense. Please let me know if you have any questions.